src/dotnet/Program.cs

```diff
@@ -6,26 +6,37 @@
 using System.IO;
 using System.Linq;
 using Microsoft.DotNet.Cli.Utils;
-using Microsoft.DotNet.Tools.Test;
-using Microsoft.Extensions.PlatformAbstractions;
-using NuGet.Frameworks;
 using Microsoft.DotNet.ProjectModel.Server;
 using Microsoft.DotNet.Tools.Build;
 using Microsoft.DotNet.Tools.Compiler;
 using Microsoft.DotNet.Tools.Compiler.Csc;
-using Microsoft.DotNet.Tools.Compiler.Native;
 using Microsoft.DotNet.Tools.Help;
 using Microsoft.DotNet.Tools.New;
 using Microsoft.DotNet.Tools.Publish;
-using Microsoft.DotNet.Tools.Repl;
-using Microsoft.DotNet.Tools.Resgen;
 using Microsoft.DotNet.Tools.Restore;
 using Microsoft.DotNet.Tools.Run;
+using Microsoft.DotNet.Tools.Test;
+using Microsoft.Extensions.PlatformAbstractions;
+using NuGet.Frameworks;
 
 namespace Microsoft.DotNet.Cli
 {
     public class Program
     {
+        private static Dictionary<string, Func<string[], int>> s_builtIns = new Dictionary<string, Func<string[], int>>
+        {
+            ["build"] = BuildCommand.Run,
+            ["compile-csc"] = CompileCscCommand.Run,
```

Comment:
@davidfowl removed it with https://github.com/dotnet/cli/commit/c7fe191f0bbae27f729a4b1fa183a16e59e61098 for issue #1574.